xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -174,7 +174,7 @@
 #define ADDON_INSTANCE_VERSION_VISUALIZATION_DEPENDS  "addon-instance/Visualization.h" \
                                                       "c-api/addon-instance/visualization.h"
 
-#define ADDON_INSTANCE_VERSION_VIDEOCODEC             "2.0.3"
+#define ADDON_INSTANCE_VERSION_VIDEOCODEC             "2.0.4"
 #define ADDON_INSTANCE_VERSION_VIDEOCODEC_MIN         "2.0.1"
```

Comment:
As you are *inserting* new enum values, numeric value of VIDEOCODEC_FORMAT_MAXFORMATS will be changed, thus breaking every add-on that uses this value. 

ADDON_INSTANCE_VERSION_VIDEOCODEC_MIN need to be increased and all Videocodec add-ons need to be recompiled / version-bumped / re-released. ADDON_INSTANCE_VERSION_VIDEOCODEC_MIN

Reply:
thanks i had forgot about max value

Reply:
i have not found other video binary addons that integrate a video decoder that use these includes
do you know someone?

Reply:
I only know about inputstream.adpative,  but I might be wrong on this.

Reply:
i tried search also on pvr not found uses,
sice its limited use i can also backport to nexus?

Reply:
> sice its limited use i can also backport to nexus?

Usually we do not backport breaking API changes. Not sure who can judge on this as @AlwinEsch, our binary add-on system maintainer, is currently not active.

Reply:
i know, since there will also be a braking change for the skins i thought maybe this could also be accepted
better to ask than nothing

Reply:
We just never backport API changes. The number of users it can impact is immense. Not worth the risk.

Reply:
fine, i will coordinate with glennguy for merge this and make new release of ISAdaptive